### PR TITLE
Configuration restructuring

### PR DESCRIPTION
**Pull request by DanielHettegger**
Updated the structure of the config files as described in this graphic:

grafik

### CODE REVIEW

File: imp_act/environments/__init__.py
Comment (omniscientoctopus) [inline imp_act/environments/__init__.py:1]: Can we use a loop instead?

```
presets = ["ToyExample-v1", "Montenegro-v1", "Denmark-v1", "Belgium-v1"]

for name in presets:

    # Numpy environments
    Registry().register(
        name=name,
        cls=numpy_environment_loader,
        parameters={"filename": f"{environment_path}/presets/{name}/{name}.yaml"},
    )

    # JAX environments
    Registry().register(
        name=f"{name}-jax",
        cls=jax_environment_loader,
        parameters={"filename": f"{environment_path}/presets/{name}/{name}.yaml"},
    )
```

**Review by omniscientoctopus — DISMISSED** [review]
@DanielHettegger, this looks great, thanks for making the changes so quickly!

**Comment by omniscientoctopus** [comment]
I have added the above illustration to the README in the environment directory, to serve as an overview for devs. I will add a simpflied version of the illustration in the docs as many details here are not relevant to the end user.

**Review by omniscientoctopus — APPROVED** [review]

**Comment by DanielHettegger** [comment]
Very nice, thanks Prateek!

**Pull request merged**